scripts/deploy.sh

```diff
@@ -102,6 +102,7 @@ function main() {
     "dev.grow.image=${internal_registry}/${grow_repo_image}"
     "metrics.adjusted_polling_rate=true"
     "pprof=true"
+    "prometheus.rules.enable=false"
```

Comment:
To ensure validity of the rules, it may actually be better to [deploy prometheus-operator-crds in the test env](https://artifacthub.io/packages/helm/prometheus-community/prometheus-operator-crds) and enable this flag, even though the rules won't have any effect?

Reply:
Interesting, so I guess I would have to match the `Application version` to the CRD version mentioned here.
https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack/56.13.1#from-55-x-to-56-x

If so, I am open to that change.

FWIW, here where we deploy the servicemonitor for our e2e test currently. 
https://github.com/jmcgrath207/k8s-ephemeral-storage-metrics/blob/master/scripts/create_kind.sh#L25

Reply:
Alright, I will take a look and update this tomorrow.

Reply:
Should be resolved. Please double-check.